tools/integration_tests/util/setup/setup.go

```diff
@@ -183,31 +198,24 @@ func ExecuteTest(m *testing.M) (successCode int) {
 	return successCode
 }
 
-func ExecuteTestForFlags(flags [][]string, m *testing.M) (successCode int) {
+func ExecuteTestForFlags(flags []string, m *testing.M) (successCode int) {
```

Comment:
I believe this function is running for single flag and if yes why do we need ExecuteTestForFlags & ExecuteTest ?

Reply:
This function will run for one array of flags. E.g., mentioned [here](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/tools/integration_tests/readonly/readonly_test.go#L68).

ExecuteTestForFlags will run for testBucket mounted in the directory.

It will unmount the bucket after running tests, and in case of failure, print passing flags for which mounting it's failed.

ExecuteTest is useful incase of mountedDirectory flag. which does not need the things mentioned in ExecuteTestForFlags and directly execute the tests.

Reply:
earlier ExecuteTestForFlags was used for testing sets of flags but now it is being used to test one set of flag. So we should rename it to something like ExecuteTestForFlagsSet

Reply:
fixed